HARP-2023-Summer/Mapping/Config/mapstyle_config.R

```diff
@@ -43,36 +43,48 @@ run_sets <- list(
     riverseg_metrics=list( 
       list(
         metric='l30_Qout', 
+        data_set = 'rseg',
         column_name='percentDiff_l30_Qout_runid_11_runid_13', 
         run_label = '2040 WSP L30 %', 
-        tables_cols = c('name', 'riverseg', 'Metric', 'runid_11_l30_Qout', 'runid_13_l30_Qout', 'percentDiff_l30_Qout_runid_11_runid_13'),
+        tables_cols_r = c('name', 'riverseg', 'Metric', 'runid_11_l30_Qout', 'runid_13_l30_Qout', 'percentDiff_l30_Qout_runid_11_runid_13'),
+        tables_cols_f = c('facility', 'Facility_hydroid', 'locality', 'five_yr_avg', 'runid_11_wd_mgd', 'runid_13_wd_mgd', 'wsp2020_2040_mgy'),
         sort_col = 'percentDiff_l30_Qout_runid_11_runid_13',
-        sort_decreasing = FALSE
+        sort_decreasing = FALSE, 
+        show_map = TRUE
       ),
       list(
         metric='7q10', 
+        data_set = 'rseg',
         column_name='percentDiff_7q10_runid_11_runid_13', 
         run_label = '2040 WSP 7q10 %', 
         tables_cols = c('name', 'riverseg', 'Metric', 'runid_11_7q10', 'runid_13_7q10', 'percentDiff_7q10_runid_11_runid_13'),
+        tables_cols_f = c('facility', 'Facility_hydroid', 'locality', 'five_yr_avg', 'runid_11_wd_mgd', 'runid_13_wd_mgd', 'wsp2020_2040_mgy'),
```

Comment:
@EllaF21 I think we don't need a different list variable for facility versus river columns, `table_cols` should be sufficient -- see comments below on how that implementation can be done.